src/main/java/org/jabref/gui/bibtexkeypattern/BibtexKeyPatternDialog.java

```diff
@@ -19,7 +19,7 @@ public BibtexKeyPatternDialog(BasePanel panel) {
         this.bibtexKeyPatternPanel = new BibtexKeyPatternPanel(panel);
         this.panel = panel;
         this.metaData = panel.getBibDatabaseContext().getMetaData();
-        AbstractBibtexKeyPattern keyPattern = metaData.getCiteKeyPattern(Globals.prefs.getKeyPattern());
+        AbstractBibtexKeyPattern keyPattern = metaData.getCiteKeyPattern(Globals.prefs.getGlobalBibtexKeyPattern());
```

Comment:
What do you think about renaming `BibtexKeyPattern` to `CiteKeyPattern` to be consistent with the naming in the metadata class?

Reply:
I'm really not sure about this. On one hand it would make sense, because it would abstract this preference beyond bibtex and biblatex.  On the other hand, it is closely tangled to the term bibtexKey, which has already become a fixed technical term in JabRef.